api/src/main/java/org/apache/flink/agents/api/prompt/Prompt.java

```diff
@@ -76,6 +77,14 @@ public String formatString(Map<String, String> kwargs) {
                 });
     }
 
+    public List<ChatMessage> formatMessages(String defaultRoleValue, Map<String, Object> kwargs) {
```

Comment:
Why do we need to pass the Object type as the value here? Can't we reuse the `formatMessages(MessageRole defaultRole, Map<String, String> kwargs)`?

This method is not used in Java. We should have documentation to say this is intended to be used by Python code.